app/assets/javascripts/dataTables/locales/renamer.rb

```diff
@@ -0,0 +1,12 @@
+# Get the translations from:
+# https://github.com/DataTables/Plugins/tree/master/i18n
+
+require 'i18n_data'
+
+(Dir['*'] - ['renamer.rb']).each do |f|
+  code = I18nData.language_code(File.basename(f, ".*"))
+
+  next if code.nil?
+
+  File.rename(f, code.downcase + '.lang')
```

Comment:
This is a cool script, but I wanted to add `.lang` to the files, just 'cause files with no extension look weird :)